tutorials/tmva/TMVA_CNN_Classification.C

```diff
@@ -379,7 +373,7 @@ void TMVA_CNN_Classification(std::vector<bool> opt = {1, 1, 1, 1})
       TString inputLayoutString("InputLayout=1|16|16");
 
       // Batch Layout
-      TString batchLayoutString("BatchLayout=100|1|256");
+      //TString batchLayoutString("BatchLayout=100|1|256");
```

Comment:
There are many outcommented lines in the tutorial. I would be in favor of removing them.